packages/discord.js/src/structures/Webhook.js

```diff
@@ -225,19 +241,22 @@ class Webhook {
     });
 
     const { body, files } = await messagePayload.resolveFiles();
-    const d = await this.client.rest.post(Routes.webhook(this.id, this.token), {
+    const data = await this.client.rest.post(Routes.webhook(this.id, this.token), {
       body,
       files,
       query,
       auth: false,
     });
 
-    if (!this.client.channels) return d;
-    return this.client.channels.cache.get(d.channel_id)?.messages._add(d, false) ?? new (getMessage())(this.client, d);
+    if (!this.client.channels) return data;
+    return (
+      this.client.channels.cache.get(d.channel_id)?.messages._add(data, false) ?? new (getMessage())(this.client, data)
```

Comment:
How come the linter didn't flag this? 🤔 

```suggestion
      this.client.channels.cache.get(data.channel_id)?.messages._add(data, false) ?? new (getMessage())(this.client, data)
```